src/lib.rs

```diff
@@ -212,42 +197,81 @@ impl fmt::Display for ValidationResults {
 
 impl ValidationResults {
     const fn new() -> Self {
-        Self {
-            invalid_tests: Vec::new(),
-            invalid_constants: Vec::new(),
-            invalid_scripts: Vec::new(),
-            invalid_src: Vec::new(),
-        }
+        Self { invalid_items: Vec::new() }
     }
```

Comment:
you should be able to `#[derive(Default)]` for this

Reply:
Cleaned up in 874ae2a